server/src/main/resources/schema/wildfly-config_11_0.xsd

```diff
@@ -4978,6 +4978,33 @@
                             </xs:documentation>
                         </xs:annotation>
                     </xs:enumeration>
+                    <xs:enumeration value="WildFly16.0">
```

Comment:
could you also add `EAP7.3` please?

Reply:
It doesn't make sense for EAP 7.3 (which included this schema version) to have an exclude for EAP 7.3.

Reply:
EAP 7.3 comes with wildfly-core 10.x, this is for WF19

Reply:
EAP 7.3 includes version 11 of the schema.